core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -959,7 +960,7 @@ class ZkAdminManager(val config: KafkaConfig,
         } else if (requestStatus.mechanism == Some(ScramMechanism.UNKNOWN)) {
           (requestStatus.user, unknownScramMechanismMsg)
```

Comment:
the condition above should use `.contains` instead of `== Some`

Reply:
It looks weird to me but it seems `contains()` is Scala idiomatic, so I made the change